src/styles/main.scss

```diff
@@ -1,3 +1,56 @@
-body {
-  margin: 0;
+@import "./utils/variables";
+@import "./utils/mixin";
+@import "./utils/normalize";
+
+.page {
+  display: flex;
+  justify-content: center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendar-width;
+  gap: $flex-gap;
+  padding: $calendar-pading;
+  transform: translateY(25%);
+
+  @for $day from 28 through 31 {
+    &--month-length--#{$day} > :nth-child(n +#{$day + 1}) {
+      display: none;
+    }
+  }
+
+  @each $first-day, $number in $day-of-week {
+    &--start-day--#{$first-day} :first-child {
+      margin-left: ($cell-size + $flex-gap) * $number;
+    }
+  }
+
+  &__day {
+
+    @include square($cell-size);
+    display: flex;
+    gap: $flex-gap;
+    box-sizing: border-box;
```

Comment:
Taking into account that you placed box-sizing in "*" selector, I'm not quite sure that you need it here.

Reply:
thank you!